scripts/package.js

```diff
@@ -0,0 +1,18 @@
+const process = require('process');
+const path = require('path');
+const baseDir = path.join(__dirname, '..');
+
+require('electron-packager')({
```

Comment:
Personally, I wold prefer to have `electron-packager` explicitly imported at the top of the file.